src/Components/BoxNumber.js/BoxNumber.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Container } from 'react-bootstrap';
+
+const BoxNumber = ({ textContent }) => (
+  <Container className="bg-secondary py-3 text-end">{textContent}</Container>
+);
+
+BoxNumber.propTypes = {
+  textContent: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+};
+
```

Comment:
while you did an impressive job working on the project requirements I noticed you disabled eslint . It is against micro verse policy to disable linters because linters help us write clean code. It would be nice if your enabled eslint

Reply:
Thanks for the information. I will delete the dissabling line